### PR TITLE
parse the userId

### PR DESCRIPTION
**Pull request by kdivya153**

### CODE REVIEW

**Review by kschrief — APPROVED** [review]

**Review by michaelcfine — APPROVED** [review]
👍

File: server/src/stateManagement.mjs [hunk 1]
```diff
@@ -160,6 +137,7 @@ function getUserId(userId){
   // //Checking if user or appId is present in state object
   if (!(userId in state)){
     for(var key in users){
+
```
Comment (michaelcfine) [inline server/src/stateManagement.mjs:140]: I still see uses of "~", which seems less than ideal. Once the currentUserId is parsed and each userId is parsed within the loop, I don't think there should be any string-related manipulation going on.

**Review by michaelcfine — CHANGES_REQUESTED** [review]
The parsedUserId change is good, but I still see uses of "~", etc. in the code. Once both userid values are parsed (one at the top, one in the loop), there shouldn't be a need for string inspection anymore.

File: server/src/stateManagement.mjs (hunk 1 above)
Comment (kdivya153) [reply server/src/stateManagement.mjs:140]: Thanks Mike for pointing it out,We have separate ticket to handle the user inside the loop.This ticket is only for ParsedUserId().Here is the ticket:https://ccp.sys.comcast.net/browse/SYSTEST-9362

**Review by michaelcfine — APPROVED** [review]
👍 due to ticket confusion

**Review by kschrief — APPROVED** [review]

**Pull request merged**